oidc-authentication.md

```diff
@@ -234,39 +242,54 @@ access resources using a traditional `Bearer` tokens.
 
 ## DPoP Validation
 
-If a `cnf` claim is present in the Access Token, then it must a DPoP Proof must be present and
-validated using the methods outlined in the
-[DPoP Internet-Draft](https://tools.ietf.org/html/draft-fett-oauth-dpop-04#section-4.2).
+If a `cnf` claim is present in the DPoP-bound Access Token, then the DPoP
```

Comment:
I it is not present than we have regular **Bearer** Access Token rather than **DPoP**(-bound) Access Token. I'll mark following section which I think were anticipating that Access Token may be either **DPoP**(-bound) or **Bearer**.